src/org/gk/database/AttributeEditConfig.java

```diff
@@ -158,6 +191,10 @@ public void loadConfig(Document document) {
         NodeList childNodes = root.getChildNodes();
         int size = childNodes.getLength();
         java.util.List attNameList = new ArrayList();
+        
+        // We could use the above attNameList, but having a unique list may help
+        // attributes and classes things separate.
+        java.util.List classNameList = new ArrayList();
```

Comment:
Move the above new lines inside else if (node.getNodeName().equals("uncreatableClass") { // If you don't have this element, you just waste creating an empty classNameList.

Reply:
That's a good point!